src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -447,7 +449,10 @@ def _forward(self, model_inputs, return_timestamps=False, generate_kwargs=None):
                 attention_mask=attention_mask,
                 **generate_kwargs,
             )
-            out = {"tokens": tokens}
+            if return_timestamps == "word":
+                out = {"tokens": tokens["sequences"], "token_timestamps": tokens["token_timestamps"]}
```

Comment:
This will also be done for `speecht5` and all the `seq2seq` models no? (and we don't want that)

Reply:
Ah, good catch!